docs/best-practices.md

```diff
@@ -204,7 +204,7 @@ Asciidoctor provides five admonition style labels: `NOTE`, `TIP`, `IMPORTANT`, `
 
 An `admonition` is written in following example style: `[<label>:]` Text
 
-Example: `[NOTE:]` If you have a large ownCloud installation and have shell access, you should ...
+Example: `NOTE:` If you have a large ownCloud installation and have shell access, you should ...
```

Comment:
Markdown doesn't support admonitions. What about just bolding the text?

Reply:
This is not Markdown !
We write BP for AsciiDoc 
This example is for AsciiDoc presented in Markdown :smile:

Reply:
There is such a thing?

Reply:
Yes, believe me